src/wiki/ventoy.md

```diff
@@ -0,0 +1,131 @@
+
+::: danger
+Installing Ventoy to create a bootable drive **will wipe all your data from that drive**. Back up your data on the drive before proceeding.
+:::
+1. Download Ventoy [here](https://www.ventoy.net/en/download.html). This is the tool that we will be using to create a bootable drive.
+![Download page of Gparted](img/ventoy/Gparted.png)
+2. Plug in a USB drive that has at least 8GB.
+3. Right click on the file that you have just downloaded and select extract. Then, enter the folder named `ventoy-1.0.xx` and open an application named `Ventoy2Disk.exe`. A UAC prompt will pop-up, click yes.
+4. Select a drive in the device selection section. This will be your bootable drive. If you can't see the drive you want to use, press the green button to refresh the list.
+![Select USB](<img/ventoy/Ventoy screen.png>)
+5. And the top right of the program click on options and get sure that the secure boot option is enabled and in partition stile select “GPT”.
+![Menu for selecting settings in Ventoy](<img/ventoy/Secure boot and GPT.png>)
+6. Click install, and Ventoy will create a bootable drive for you. After it finishes, you will see a drive named as "Ventoy", this is where you will place your ISO files for the bootable drive.
+![Click on Install button](img/ventoy/Install.png)
+
+## Downloading Windows to the Bootable Drive
+### Downloading the ISO file
+
+1. For Windows 10, go to [https://www.microsoft.com/software-download/windows10](https://www.microsoft.com/software-download/windows10) <br>
+To download Windows 11 (Recommended), head to [https://www.microsoft.com/software-download/windows11](https://www.microsoft.com/software-download/windows11)
+::: tip
+If you are not on Windows, you will see an option to "Select Edition". Select the proper edition, and language, and download the file. Once the download finishes, move the ISO file to the Ventoy drive.
+:::
+
+
+2. Click the **Download Tool Now** button.
+
+   ![](./img/downloading-windows/downloadtoolnow.png)
+
+3. After the tool has been downloaded, run it.
+
+### Using the tool
+
+1. The first thing you will see is this screen, click Accept.
+
+   ![](./img/downloading-windows/licenseterms.png)
+
+2. Wait for the installer to prepare the files needed.
+
+   ![](./img/downloading-windows/gettingthings.png)
+
+3. This is where you select what you want to do.
+
+Select **Create installation media for another PC**, then press next.
+
+    ![](./img/downloading-windows/whatdoyouwanttodo.png)
+
+4. Use the recommended settings, or select your preferred Language, Edition, and Architecture.
+
+::: tip Note
+If you're planning to install Windows on another PC and don't know what architecture it uses, then select "Both".
+:::
+
+   ![](./img/downloading-windows/changelanguagearchitecture.png)
+
+5. Select **ISO file**, then press next.
+
+![](./img/downloading-windows/mediatouse.png)
+
+6. Select a path to save to.
+### Microsoft Defender Offline
```

Comment:
This section starts abruptly and seems to carry on from the previous section, please fix this